### PR TITLE
Added component/tests for Upload Grade History Job

### PR DESCRIPTION
**Pull request by ericknee**
Fixes Issue #18 
Under the Manage Jobs tab, the Update Grade Info now has a button to upload the grade history.

Before:
Screenshot 2024-05-29 at 5 39 38 PM

After:
Screenshot 2024-05-29 at 5 38 59 PM

Dokku Deployment: https://project-ericknee-dev.dokku-01.cs.ucsb.edu/admin/jobs

### CODE REVIEW

**Review by phamaiden — APPROVED** [review]
LGTM

**Review by monicatrinh1 — APPROVED** [review]
LGTM!

File: frontend/src/tests/pages/AdminJobsPage.test.js [hunk 1]
```diff
@@ -221,4 +221,36 @@ describe("AdminJobsPage tests", () => {
...
+
+    expect(axiosMock.history.post[0].url).toBe(
+      "/api/jobs/launch/uploadGradeData",
+    );
+
+    // const expectedKey = "BasicSearch.Subject-option-ANTH";
+    // await waitFor(() =>
+    //   expect(screen.getByTestId(expectedKey).toBeInTheDocument),
+    // );
+  });
```
Comment (pconrad) [inline frontend/src/tests/pages/AdminJobsPage.test.js:255]: Commented out code

**Review by pconrad — CHANGES_REQUESTED** [review]
Commented out code

File: frontend/src/tests/pages/AdminJobsPage.test.js (hunk 1 above)
Comment (ericknee) [reply frontend/src/tests/pages/AdminJobsPage.test.js]: Fixed

**Pull request merged**